velox/functions/prestosql/aggregates/tests/MergeAggregateTest.cpp

```diff
@@ -136,35 +212,126 @@ TEST_F(MergeAggregateTest, mergeTDigestOneNullMatchJava) {
+      result.value<TypeKind::VARCHAR>(), expectedMergedQDigest));
+}
+
+TEST_F(MergeAggregateTest, mergeQDigestOneNullMatchJava) {
+  registerQDigestType();
+  // Base64-encoded QDigests from Presto Java
+  // QDigest: values 1 to 50
+  const std::string kQDigest1 =
+      "AK5H4XoUru8/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAPA/AAAAAAAASUBjAAAAAAAAAAAAAPA/AAAAAAAA8L8AAAAAAAAA8D8AAAAAAAAAwAAAAAAAAADwPwAAAAAAAAjAzwAAAAAAAAAAAAAAAAAAAMAAAAAAAAAA8D8AAAAAAAAQwAAAAAAAAADwPwAAAAAAABTAywAAAAAAAAAAAAAAAAAAEMAAAAAAAAAA8D8AAAAAAAAYwAAAAAAAAADwPwAAAAAAABzAywAAAAAAAAAAAAAAAAAAGMDPAAAAAAAAAAAAAAAAAAAQwNMAAAAAAAAAAAAAAAAAAADAAAAAAAAAAPA/AAAAAAAAIMAAAAAAAAAA8D8AAAAAAAAiwMcAAAAAAAAAAAAAAAAAACDAAAAAAAAAAPA/AAAAAAAAJMAAAAAAAAAA8D8AAAAAAAAmwMcAAAAAAAAAAAAAAAAAACTAywAAAAAAAAAAAAAAAAAAIMAAAAAAAAAA8D8AAAAAAAAowAAAAAAAAADwPwAAAAAAACrAxwAAAAAAAAAAAAAAAAAAKMAAAAAAAAAA8D8AAAAAAAAswAAAAAAAAADwPwAAAAAAAC7AxwAAAAAAAAAAAAAAAAAALMDLAAAAAAAAAAAAAAAAAAAowM8AAAAAAAAAAAAAAAAAACDAAAAAAAAAAPA/AAAAAAAAMMAAAAAAAAAA8D8AAAAAAAAxwMMAAAAAAAAAAAAAAAAAADDAAAAAAAAAAPA/AAAAAAAAMsAAAAAAAAAA8D8AAAAAAAAzwMMAAAAAAAAAAAAAAAAAADLAxwAAAAAAAAAAAAAAAAAAMMAAAAAAAAAA8D8AAAAAAAA0wAAAAAAAAADwPwAAAAAAADXAwwAAAAAAAAAAAAAAAAAANMAAAAAAAAAA8D8AAAAAAAA2wAAAAAAAAADwPwAAAAAAADfAwwAAAAAAAAAAAAAAAAAANsDHAAAAAAAAAAAAAAAAAAA0wMsAAAAAAAAAAAAAAAAAADDAAAAAAAAAAPA/AAAAAAAAOMAAAAAAAAAA8D8AAAAAAAA5wMMAAAAAAAAAAAAAAAAAADjAAAAAAAAAAPA/AAAAAAAAOsAAAAAAAAAA8D8AAAAAAAA7wMMAAAAAAAAAAAAAAAAAADrAxwAAAAAAAAAAAAAAAAAAOMAAAAAAAAAA8D8AAAAAAAA8wAAAAAAAAADwPwAAAAAAAD3AwwAAAAAAAAAAAAAAAAAAPMAAAAAAAAAA8D8AAAAAAAA+wAAAAAAAAADwPwAAAAAAAD/AwwAAAAAAAAAAAAAAAAAAPsDHAAAAAAAAAAAAAAAAAAA8wMsAAAAAAAAAAAAAAAAAADjAzwAAAAAAAAAAAAAAAAAAMMDTAAAAAAAAAAAAAAAAAAAgwNcAAAAAAAAAAAAAAAAAAADAAAAAAAAAAPA/AAAAAAAAQMAAAAAAAAAA8D8AAAAAAIBAwL8AAAAAAAAAAAAAAAAAAEDAAAAAAAAAAPA/AAAAAAAAQcAAAAAAAAAA8D8AAAAAAIBBwL8AAAAAAAAAAAAAAAAAAEHAwwAAAAAAAAAAAAAAAAAAQMAAAAAAAAAA8D8AAAAAAABCwAAAAAAAAADwPwAAAAAAgELAvwAAAAAAAAAAAAAAAAAAQsAAAAAAAAAA8D8AAAAAAABDwAAAAAAAAADwPwAAAAAAgEPAvwAAAAAAAAAAAAAAAAAAQ8DDAAAAAAAAAAAAAAAAAABCwMcAAAAAAAAAAAAAAAAAAEDAAAAAAAAAAPA/AAAAAAAARMAAAAAAAAAA8D8AAAAAAIBEwL8AAAAAAAAAAAAAAAAAAETAAAAAAAAAAPA/AAAAAAAARcAAAAAAAAAA8D8AAAAAAIBFwL8AAAAAAAAAAAAAAAAAAEXAwwAAAAAAAAAAAAAAAAAARMAAAAAAAAAA8D8AAAAAAABGwAAAAAAAAADwPwAAAAAAgEbAvwAAAAAAAAAAAAAAAAAARsAAAAAAAAAA8D8AAAAAAABHwAAAAAAAAADwPwAAAAAAgEfAvwAAAAAAAAAAAAAAAAAAR8DDAAAAAAAAAAAAAAAAAABGwMcAAAAAAAAAAAAAAAAAAETAywAAAAAAAAAAAAAAAAAAQMAAAAAAAAAA8D8AAAAAAABIwAAAAAAAAADwPwAAAAAAgEjAvwAAAAAAAAAAAAAAAAAASMAAAAAAAAAA8D8AAAAAAABJwMMAAAAAAAAAAAAAAAAAAEjAzwAAAAAAAAAAAAAAAAAAQMDbAAAAAAAAAAAAAAAAAAAAwPsAAAAAAAAAAAAAAAAAAPC/";
+  // Null QDigest
+  // Null QDigest
+  const std::string kQDigest2;
+
+  // Create QDigest vectors from base64 strings
+  auto qdigestData = makeFlatVector<std::string>({kQDigest1, kQDigest2});
+
+  // Create plan and execute
+  auto op =
+      createMergePlan<int64_t>(makeRowVector({qdigestData}), QDIGEST(BIGINT()));
+  auto result = readSingleValue(op);
+
+  // Expected merged QDigest base64 string from Java
```

Comment:
Is this the same as kQDigest1? Or by which SQL query do you get this expected result from Java?

Reply:
If it's the same, could we just use kQDigest1 in qdigestEquals, or use  `auto expectedMergedQDigest = kQDigest1` to avoid defining two variables with the same value?

Reply:
Updated to using kQDigest1 to keep it simple